curriculum/outlines/Linear DS.md

```diff
@@ -0,0 +1,39 @@
+# Linear DS
+---
+## Outline
+1. Static Array
+	 1. 1D Static Array
+	 2. 2D Static Array
+	 3. 3D Static Array
+2. Dynamically-Resizeable Array
+3. BitSet
+4. Bitmasks (Basic bit manipulation)
+	  * Example: Flipping i-th bit in an integer.
+5. LinkedLists
+6. Stacks
+7. Queues
+8. Dequeues
+    
+---
+## Problem Sets
```

Comment:
Did you include all problems from the original plan?

Reply:
Not all of them, A lot of problems from the original plan were implementation problems where you don't need to use arrays. However, the good ones are added.

Reply:
Please add a comment to the pull request with all problems removed with reasons.

Reply:
Ok